src/index.ts

```diff
@@ -8,6 +8,8 @@
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
 
+/// <reference path="../custom_typings/main.d.ts" />
```

Comment:
Agreed, I've figured out DefinitelyTyped enough to get my PRs accepted. I'm putting together a PR for estree right now.